lib/screen/feralfile_series/feralfile_series_page.dart

```diff
@@ -27,13 +30,42 @@ class FeralFileSeriesPage extends StatefulWidget {
 class _FeralFileSeriesPageState extends State<FeralFileSeriesPage> {
   late final FeralFileSeriesBloc _feralFileSeriesBloc;
   final _canvasDeviceBloc = injector.get<CanvasDeviceBloc>();
+  final PagingController<int, Artwork> _pagingController =
+      PagingController(firstPageKey: 0);
+  static const _pageSize = 300;
 
   @override
   void initState() {
     super.initState();
     _feralFileSeriesBloc = context.read<FeralFileSeriesBloc>();
     _feralFileSeriesBloc.add(FeralFileSeriesGetSeriesEvent(
         widget.payload.seriesId, widget.payload.exhibitionId));
+    _pagingController.addPageRequestListener((pageKey) async {
+      await _fetchPage(pageKey);
+    });
+  }
+
+  Future<void> _fetchPage(int pageKey) async {
+    try {
+      final newItems = await injector<FeralFileService>().getSeriesArtworks(
+          widget.payload.seriesId, widget.payload.exhibitionId,
+          withSeries: true, offset: pageKey);
+      final isLastPage = !newItems.paging.shouldLoadMore;
+      if (isLastPage) {
+        _pagingController.appendLastPage(newItems.result);
+      } else {
+        final nextPageKey = pageKey + _pageSize;
```

Comment:
Pass _pageSize as litmit of `getSeriesArtworks` function or use that function response to calculate `nextPageKey`

Reply:
Since they share the same value, lint warns to remove it

Reply:
So we should use the response to calculate `nextPageKey` because the value of `nextPageKey` is wrong if the `_pageSize` is different with the `default limit` value. `_pageSize` and `default limit` are not the same variable, they just have the same value.